fr/orm/saving-data.rst

```diff
@@ -437,7 +443,7 @@ Par exemple, considérons le cas suivant::
             ['body' => 'A new comment'],
         ]
     ];
-    $articles->patchEntity($entity, $newData);
+    $articles->patchEntity($article, $newData);
     $articles->save($article);
```

Comment:
changing $article here by $entity would also work.
Anyway, would you be able to update also the original example?